src/components/mobile/ExploreMobile.tsx

```diff
@@ -24,8 +28,12 @@ export default function ExploreMobile() {
     setShowSummary(showSummary);
   }
 
+  useEffect(() => {
+    setShowFilters(pulsateUnityFilter);
+  }, [pulsateUnityFilter])
+
   return (
-    <div className="fill flex">
+    <div className="fill flex" style={{height: "100%"}}>
```

Comment:
Also using inline here for the same reason as pubs PR (changes in SASS will automatically negate this)